botx/bot/handler_collector.py

```diff
@@ -44,13 +50,38 @@ def __init__(self, middlewares: Optional[Sequence[Middleware]] = None) -> None:
             SystemEventHandlerFunc,
         ] = {}
         self._middlewares = self._reversed_middlewares(middlewares)
+        self._tasks: "WeakSet[asyncio.Task[None]]" = WeakSet()
 
     def include(self, *others: "HandlerCollector") -> None:
         """Include other `HandlerCollector`."""
 
         for collector in others:
             self._include_collector(collector)
 
+    def insert_exception_middleware(
```

Comment:
Pls place this method before private ones. It used only by bot, no library users.